src/nvim/option.c

```diff
@@ -454,6 +454,9 @@ void set_init_1(void)
   /* Set all options to their Vim default */
   set_options_default(OPT_FREE);
 
+  // set 'laststatus'
+  last_status(false);
```

Comment:
`set_options_default()` is not enough. One sees this by (on master) `set ls=2` and then `set all&` (which calls `set_options_default()`) . Now `set ls` shows 1 but nvim still behaves _as if_ `&ls=2`.